system-addon/test/functional/mochitest/browser.ini

```diff
@@ -1,9 +1,5 @@
 [DEFAULT]
-# XXX This defaults to forcing activity-stream tests to be skipped in m-c,
-# since, as of this writing, mozilla-central itself is still turned off.
-# The tests can be run locally using 'npm run mochitest' which does various
-# overrides.
-skip-if=!activity_stream
+skip-if=!nightly_build
```

Comment:
Why this?  Because we haven't been approved to uplift yet?

Whatever the reason, I think this wants a comment in the ini file about why.